### PR TITLE
Refactored to be installable as a Django App

### PR DESCRIPTION
**Pull request by kave**
### Updated
- included settings file
- converted project to be pip installable 
- updated readme
- reformatted wsdl file 
- updated client to handle Wagtail Publishing & maintain client purging of cache data
### Review

@rosskarchner

### CODE REVIEW

File: publish_eccu/PublishECCU.xml [hunk 1]
```diff
@@ -1,584 +1,241 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (kave) [inline publish_eccu/PublishECCU.xml:1]: just formatting

**Comment by acdha** [comment]
Do you think it's worth moving the `argparse` bits in publish.py into `main` so anyone calling it as a library wouldn't execute them? That seems a lot more likely after packing it for PyPI.

**Comment by kave** [comment]
@acdha thats very true, i'll make that change. thanks!

**Comment by rosskarchner** [comment]
Does it continue to work the old way, or is this only usable within Django now?

**Comment by kave** [comment]
@rosskarchner yea it works both ways

**Comment by rosskarchner** [comment]
👍

**Comment by rosskarchner** [comment]
(also, thanks for chiming in, @acdha )

**Comment by acdha** [comment]
Thanks for releasing this! I've been meaning to tackle this for years but it hasn't been enough of a priority to get over the too-much-XML hurdle 😀

**Pull request merged**